src/main/runner.ts

```diff
@@ -190,7 +190,14 @@ export default class RunnerMain extends Runner {
               || this.findForgottenTestById(data.id);
             assert(test, `Couldn't find test by id: ${data.id}`);
 
-            this.emit(event, test, data.err);
+            const testErrors = (test as any).err;
+
+            ((typeof(testErrors) === 'object')
+              && (testErrors !== null)
+                && Object.keys(testErrors).length > 0)
```

Comment:
This is very minor but please use simple if..else instead of ternary here: it makes the code much easier to understand.

Reply:
@1999 will fix